cmd/summaraizer-slack/main.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"net/http"
+
+	"github.com/ioki-mobility/summaraizer-slack/api"
+)
+
+func main() {
+	http.HandleFunc("/", api.Handler)
```

Comment:
I would suggest also adding a simple health endpoint! Makes it easier to deploy in a containerized environment because then the orchestrator know exactly when your application is down/has issues

Reply:
Sure thing!
I just chatted with Haiku 3.5 about that as well, what would you actually expect to do in the health endpoint?
I would definitely check if all env variables are set correctly.
But should I also check the validity of those? So should I also do an api call to slack and ollama or openai to check if everything is fine? 🤔
Or would this be too much?

What is your take on this?

Reply:
My personal take for best support in container based environments would be to serve on two ports:

Your normal application port e.g. 8080 and another port for metrics and health checks e.g 9090

on 9090 I would serve these endpoints:

- /healthz that just responds with 200 / a simple health check to see if the app is active 
- /ready that does some more intensive but meaningful checks (like checking the ollama backend connection etc)
- /metrics for whatever metrics you want to provide 


With the port separation an admin could just forward the 8080 port to the public but keep 9090 internal. This removes the need for any additional with/token checks for the health endpoint

Reply:
For a first quick an nice solution I guess it would be also fine to use the same port with 

- /healthz for a quick health check
- /health or /ready for a meaningful health check 

admins can the use a reverse proxy to forbid /health to the public or use a token for auth. That part should not be done in the go application

Reply:
Wow! Thanks @zadjadr for that detailed explanation! 

I added a healthz endpoint [here](https://github.com/ioki-mobility/summaraizer-slack/pull/8/commits/781526005914c10a436f1f3720f75abff09cb440) and a ready endpoint [here](https://github.com/ioki-mobility/summaraizer-slack/pull/8/commits/d4544d6202da14739eb25b379b1d90dc08e9bd9e).
For now the ready endpoint is just checking Slack.
I will add some more stuff to it later, when I also tackle #1